packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# shellcheck disable=SC1090
+set -eo pipefail
+
+export resourceGroupName=""
+export functionAppName
+export resourceName
+
+exitWithUsageInfo() {
+    echo "
+Usage: $0 -r <resource group>
+"
+    exit 1
+}
+
+# Set default ARM Function App template files
+templateFilePath="${0%/*}/../templates/function-app-template.json"
+
+# Read script arguments
+while getopts "r:"  option; do
+    case $option in
+		r) resourceGroupName=${OPTARG} ;;
+		*) exitWithUsageInfo ;;
+		esac
+done
+
+if [ -z "$resourceGroupName" ]; then
+	echo "Either one of subscription, resourceGroupName or location is empty"
```

Comment:
misleading message, only rg is required as parameter. Either add to options rest of params or update the message